app/jobs/application_job.rb

```diff
@@ -1,4 +1,6 @@
 class ApplicationJob < ActiveJob::Base
+  include Foreman::Observable
```

Comment:
`Foreman::Observable` is already included in `::Foreman::ObservableJob`. Unless I'm missing something having it here isn't necessary

Reply:
Yeah an overlook.